index.html

```diff
@@ -196,5 +196,47 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+
+    <footer>
+        <div class="icons">
+        <p class="attract">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            Full name
+            Email address
+            Enter text here
+            Get in touch
+        </p>
+        <form action="https://formspree.io/f/xeqdogov" method="post">
+                <div>
+                    <input type="text" name="fullname" placeholder="Full name" maxlength="30" required class="formel">
+                </div>
+                <div>
+                    <input type="email" name="email" placeholder="Email address" required class="formel">
+                </div>
+                <div>
+                   
+                
+                <textarea  name="message" maxlength="500" class="textarea" placeholder="Enter your text" required>
+                </textarea>
+                </div>
+
```

Comment:
### Your TextArea
![student-textarea (copy)](https://user-images.githubusercontent.com/93039359/186458018-28af2587-e87e-4605-9f92-7ce4b082e844.png)

### Figma TextArea
![text-area-figma](https://user-images.githubusercontent.com/93039359/186458038-efc8a821-2102-47f6-b60c-d8010c4cb823.png)

- [ ] Your text area is not very similar to the Figma template. Please make it similar to the figma template to make your form look professional :blush:
- [ ] When i click on these then it starts writing from the red line. Please fix it :blush: